src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/javadoc/missingjavadocmethod/InputMissingJavadocMethodInterfacePrivateMethod.java

```diff
@@ -0,0 +1,11 @@
+//non-compiled with javac: Compilable with Java9
+// private method in interface
+
+package com.puppycrawl.tools.checkstyle.checks.javadoc.missingjavadocmethod;
+
+public interface InputMissingJavadocMethodInterfacePrivateMethod {
+
+    private static void myPrivateMethod() {
```

Comment:
Please add test case with new violation that is found by regression report https://strkkk.github.io/checkstyle/8266/checkstyle/index.html#A3

Reply:
done